doc/swish/osi.tex

```diff
@@ -32,33 +32,42 @@
 \section {Theory of Operation}
 
 The operating system interface is written in C99~\cite{c99} as a
-shared library that is statically linked against the libuv and SQLite
-libraries and dynamically linked against single-threaded Chez Scheme
-9.5.  Please refer to Chapter~4 of the \emph{Chez Scheme Version 9
-  User's Guide}~\cite{chez-scheme-users-guide} for information on the
-foreign function interface. C++ is not used because C++ destructors
-may interact badly with setjmp/longjmp, used by Chez Scheme.
+shared library that links to the single-threaded Chez Scheme 9, libuv,
```

Comment:
Suggestion: delete "single-threaded" here since: (1) threading model is covered in the next paragraph and (2) it appears to imply single-threaded libuv and SQLite where the original text was careful to restrict that modifier to Chez Scheme.

Suggestion: delete the "9" here since we don't mention libuv and SQLite versions here and the reference to Chapter 4 of CSUG contains the version number.

So: "... links to the Chez Scheme, libuv, and SQLite libraries."

Reply:
Good ideas, thanks!